.circleci/config.yml

```diff
@@ -3,20 +3,58 @@ executors:
   default:
     docker:
       - image: circleci/node:12.5.0
+
 commands:
   update-npm:
+    description: 'Install latest npm'
     steps:
       - run: sudo npm install -g npm@latest
   install-npm-wee:
+    description: 'install dependencies'
     steps:
       - run: npm ci
   build:
+    description: 'Compile typescript code'
     steps:
       - run: npm run tsc
+  run-tests:
```

Comment:
Hmm all of the things under `commands` should be collection of tasks but now all of them have just one. So for example this run-tests move it to under `steps` like this.

```yml
- run:
  name: Run tests
  command: npm run test
```

This also increases readability since reader doesn't have to jump between lines 👍